src/components/Navigation.vue

```diff
@@ -2,28 +2,36 @@
   <nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
     <div class="navbar-brand">
       <router-link class="navbar-item" to="/">
-        <img src="../assets/logo.png" class="logo-nav">
+        <img src="../assets/logo.png" class="logo-nav" alt="Logo">
       </router-link>
-      <a v-on:click="onBurgerClick" v-bind:class="{'is-active': isBurgerActive}"
-         role="button" class="navbar-burger" aria-label="menu"
-         aria-expanded="false" data-target="navbarBasicExample">
-        <span aria-hidden="true"></span>
-        <span aria-hidden="true"></span>
-        <span aria-hidden="true"></span>
+      <a
+        v-on:click="onBurgerClick"
+        @keydown="onBurgerClick"
+        v-bind:class="{ 'is-active': isBurgerActive }"
+        role="button"
+        class="navbar-burger"
+        aria-label="menu"
+        aria-expanded="false"
+        data-target="navbarBasicExample"
+        href="#">
```

Comment:
Please test if this has no side-effects

Reply:
It did cause some "flickering" of the url when clicked on mobile. Probably caused by resetting the URL and then immediately setting it again from localStorage.
I removed it, but did have to add a tabindex, according to the linter.
I went with tabindex="0", since it is the only place we are using one right now.